### ISSUE TITLE
Invalid configuration

### ISSUE THREAD

**Issue by mrmckeb** [brigade/scss-lint#676]
Hi there,

I'm seeing the following error:
`Invalid configuration: (<unknown>): did not find expected key while parsing a block mapping at line 4 column 1`

It works fine if I remove the `exclude` section and `linters` keyword from the config. Top of config is below:
```
exclude:
 - 'www/latest/scss/base/_reset.scss'
 - 'www/latest/scss/base/_normalize.scss'

linters:
  BorderZero:
    enabled: true

  CapitalizationInSelector:
    enabled: true

  ColorKeyword:
    enabled: true

  Comment:
    enabled: true
```

It doesn't throw that error I use the below only:
```
  BorderZero:
    enabled: true

  CapitalizationInSelector:
    enabled: true

  ColorKeyword:
    enabled: true

  Comment:
    enabled: true
```

**Comment by tdooner** [comment]
Hi @mrmckeb, I'm not able to reproduce this with Ruby 2.2.2. Can you give me the output of `ruby --version` and perhaps the exact stacktrace for the error message?

**Comment by tdooner** [comment]
By the way, I noticed in your example that you indented the exclude items by one space and everything else by two spaces. I'm not sure if that matters, but you may consider trying a two-space indent to see if that solves the problem.

**Comment by mrmckeb** [comment]
Thanks @tdooner, you were spot on: indentation issues throughout the file. It's now working fine.

I'm not sure why the author chose different indentation styles, it's quite strange - but I also assumed the whitespace was for readability, which I'll watch out for in future.

Again, thanks for helping to resolve this.

**Comment by tdooner** [comment]
Glad to hear it's something simple. It usually is :). Cheers!
